jujugui/static/gui/src/app/components/copy-to-clipboard/copy-to-clipboard.js

```diff
@@ -39,13 +39,15 @@ class CopyToClipboard extends React.Component {
           ref="input"
           type="text"
           value={this.props.value} />
-        <button
-          className={className + '__btn'}
-          ref="btn">
-          <SvgIcon
-            name="copy-to-clipboard-16"
-            size="16" />
-        </button>
+        <span className="v1">
```

Comment:
Some components need to be under the `v1` namespace so they pick up the new classes (which in the GUI are loaded under the `.v1` namespace).